.github/workflows/internal-build.yml

```diff
@@ -2,8 +2,15 @@ name: Run internal build
 
 on:
   pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
What PR actions exactly are we excluding here? The commit message suggests that it's draft PRs but are there more? I find it useful to have CI run on draft PRs – when I open a PR with experimental changes I might not have run all tests locally and I have no idea if it works on Windows, I'd want to know if CI passes before marking the PR as ready and having reviewers look at it.

Reply:
My goal was to remove draft PRs. Personally, whenever I open a draft PR I expect it to not work and just open to share the code with someone, but it's open to discussion I guess. If you prefer this way, I don't have any strong opinions and just remove this commit.

Reply:
Yeah, I think that would be better. As a specific example, the Bazel 8 update PR is in the draft stage, I've been able to push to it and see Windows CI fail a number of times there. If I had to make it a non-draft PR to see if CI passes, that would cause lots of spam for reviewers and falsely suggest that it is ready for review.

Reply:
Strong agree with Felix here, draft PRs should have CI running by default.

Reply:
Removed this change.